CHANGELOG.md

```diff
@@ -42,7 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - FP-939: Display shared workspace description whitespace. (#435)
 - FP-997, FP-1041, FP-1042: Improve apps, add small queue support, and update execution systems when needed (#426)
 - FP-573: Prevent submitting unlicensed app (#438)
-- FP-1009: Apply ReadMore UI pattern to long Shared Workspace descriptions (#442)
+- FP-1009: Apply ShowMore UI pattern to long Shared Workspace descriptions (#442)
```

Comment:
We can keep the Changelog as it is. A future update to the changelog will note the change from ReadMore to ShowMore